src/defineFunction.js

```diff
@@ -0,0 +1,122 @@
+import functions from "./functions";
+import {groupTypes as htmlGroupTypes} from "./buildHTML";
+import {groupTypes as mathmlGroupTypes} from "./buildMathML";
+
+/* This file contains a list of functions that we parse, identified by
```

Comment:
I was thinking that `defineFunction` would be the only shared function, but I see that we also have `ordargument` and we may have other helpers in the future so keeping these functions in `function.js` and `export`ing each of them makes sense.